steps/monitor_dependabot_activity.yml

```diff
@@ -14,9 +14,9 @@ startFlow:
         messages:
           - text: Alright, the setup is done, but the journey doesn't end here.
           - text: Now, it's time to watch for Dependabot-generated pull requests. Keep an
-              eye on the [Actions Tab](${user.repository}/actions/workflows/dependabot/dependabot-updates) and check it's progress.
+              eye on the [Actions Tab](${user.repository}/actions/workflows/dependabot/dependabot-updates) and check its progress.
           - text: While the Dependabot action is running (might take a few minutes to complete) it will create a bunch of pull requests as it goes.
-          - text: Keep an eye on [your Pull Requests tab](${user.repository}/pulls) as the dependabot creates them.
+          - text: Keep an eye on [your Pull Requests tab](${user.repository}/pulls) as dependabot creates them.
```

Comment:
Lines 18-19 look like a repetition of 16-17. Do we need all of these?

Reply:
Its two different things, you check the updates that the workflow is running, the workflow creates pull requests, you need to first see that its running and while its running look at the PR tab to see the PR's, when it started running I myself did not know that its async opening PR's, I waited 3 minutes for it to finish, I rather be explicit in telling the user its opening PR's as its running so open your pr tab and find one.

Reply:
Ok, that can explain 16 and 18, but 17 and 19 still sounds like duplicates to me